ffi/tests/ffi_export.rs

```diff
@@ -296,3 +311,20 @@ fn return_result() {
         assert_eq!(42, output.assume_init());
     }
 }
+
+#[test]
+fn invoke_trait_method() {
+    let ffi_struct = get_new_struct_with_params();
+    let mut output = MaybeUninit::uninit();
```

Comment:
```suggestion
    let mut output = MaybeUninit::<Name>::new();
```

Please try to use exact types where ever possible so that we are sure this works correctly. If you can use exact types in other types as well. Initially, I did this is `valid.rs` but that file will most likely be removed because it's just a duplicate of integration tests.

Also, don't use `uninit` in tests, instead use `new`. This is just a precaution to ensure that if the feature tested is not implemented correctly it does not introduce UB because UB could make the test pass

Reply:
> does not introduce UB because UB could make the test pass
good point, thanks

Reply:
> I would try to use exact types where ever possible so that we are sure this works correctly.

This advice was ill given actually because you'll always be comparing to some actual value in `assert_eq`

Reply:
> > I would try to use exact types where ever possible so that we are sure this works correctly.
> 
> This advice was ill given actually because you'll always be comparing to some actual value in `assert_eq`

You want to rollback this change?

Reply:
No, I don't care